<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Spectrometer - UrsaPQ Remote</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
	background-color: #FFFFFF;
}

QLabel {
	font-size: 12.5pt;
}

QGroupBox {
	background-color: #EFEFEF;
    border: 1px solid gray;
    border-color: #003959;
    border-radius: 10px;
}

QGroupBox::title {
    subcontrol-position: top left;
 	border-top-left-radius: 10px;
 	border-bottom-right-radius: 10px;
    padding: 5px 20px;
    background-color: #003959;
    color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>391</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Flight Tube</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>45</y>
       <width>46</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="tofEnableBox"/>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>45</y>
       <width>166</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HV Enable</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>120</y>
       <width>361</width>
       <height>211</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Retarder</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>106</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Middle</string>
      </property>
     </widget>
     <widget class="QLabel" name="tofRetarder_act">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="tofMiddle_act">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tofSetButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>165</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set values</string>
      </property>
     </widget>
     <widget class="QLabel" name="tofRetarder_set">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>0</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="tofMiddle_set">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>30</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tofRetarder_in">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tofMiddle_in">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>106</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Mesh</string>
      </property>
     </widget>
     <widget class="QLabel" name="tofMesh_act">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="tofMesh_set">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>60</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tofMesh_in">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>106</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Extractor</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>106</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Drift</string>
      </property>
     </widget>
     <widget class="QLabel" name="tofExtractor_act">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="tofExtractor_set">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>90</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tofExtractor_in">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>90</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="tofDrift_act">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>120</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="tofDrift_set">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>120</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="tofDrift_in">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>120</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Actual</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Setpoint</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>90</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>TOF [V]</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>360</y>
      <width>391</width>
      <height>316</height>
     </rect>
    </property>
    <property name="title">
     <string>Detector</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>45</y>
       <width>46</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="mcpEnableBox"/>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>45</y>
       <width>166</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>HV Enable</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>120</y>
       <width>361</width>
       <height>181</height>
      </rect>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Front</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>106</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Phosphor</string>
      </property>
     </widget>
     <widget class="QLabel" name="mcpFront_act">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>0</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="mcpBack_act">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>30</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="mcpPhos_act">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="mcpFront_in">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>0</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="mcpSetButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>150</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Set values</string>
      </property>
     </widget>
     <widget class="QLabel" name="mcpFront_set">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>0</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="mcpBack_set">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>30</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="mcpPhos_set">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>60</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="mcpBack_in">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>30</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="mcpPhos_in">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>105</y>
        <width>106</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>iTOF MCP</string>
      </property>
     </widget>
     <widget class="QLabel" name="mcpItof_act">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>105</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QLabel" name="mcpItof_set">
      <property name="geometry">
       <rect>
        <x>195</x>
        <y>105</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>---</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="mcpItof_in">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>105</y>
        <width>76</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>90</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Actual</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>90</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Setpoint</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>90</y>
       <width>76</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>MCP [V]</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
